include/psa/crypto_struct.h

```diff
@@ -228,11 +228,11 @@ typedef struct
 #if defined(MBEDTLS_MD_C)
 typedef enum
 {
-    TLS12_PRF_STATE_INIT,       /* no input provided */
-    TLS12_PRF_STATE_SEED_SET,   /* seed has been set */
-    TLS12_PRF_STATE_KEY_SET,    /* key has been set */
-    TLS12_PRF_STATE_LABEL_SET,  /* label has been set */
-    TLS12_PRF_STATE_OUTPUT      /* output has been started */
+    PSA_TLS12_PRF_STATE_INIT,       /* no input provided */
```

Comment:
Just wondering here why not MBEDTLS_PSA_ as this is specific to the Mbed TLS PSA implementation.

Reply:
We've been inconsistent about whether to use an `MBEDTLS` prefix on things that are specific to the PSA Crypto Reference Implementation (formerly known as Mbed Crypto and now as Mbed TLS) and apply neither to other implementations or to classic Mbed TLS APIs. I personally think that using an `MBEDTLS_` prefix would actually be a good idea, but given the plan to spin off the PSA Crypto reference implementation as its own product, having `MBEDTLS_` prefixes in that is counter-productive.

In any case, in this specific instance, `check-names.sh` doesn't like it and I don't think pacifying it would be the best use of our time right now.

Reply:
ok, thanks for explaining the context. That's fine by me.